integration/it-integration/src/main/java/se/inera/intyg/rehabstod/integration/it/dto/SickLeavesRequestDTO.java

```diff
@@ -41,4 +41,5 @@ public class SickLeavesRequestDTO {
     private String protectedPersonFilterId;
     private LocalDate fromSickLeaveEndDate;
     private LocalDate toSickLeaveEndDate;
+    private List<String> rekoStatuses;
```

Comment:
Suggest change of name to rekoStatusTypeIds. This change should be made in necessary places.